src/wrapper/simulation_objects.py

```diff
@@ -915,7 +922,7 @@ def add_subscriptions(self) -> List[int]:
     @staticmethod
     def from_simulation(
         simulation_object, updater: "SimulationObjectUpdatingComponent"
-    ) -> "Train":
+    ) -> None:
         # Nothing to do (we dont load trains from the sim)
         pass
```

Comment:
So this method could be removed? Or is this planned for the future?

Reply:
Neither. The method is overridden from the abstract class. If we would remove this we would get
1. Linter errors
2. Actual errors because the abstract base throws a `NotImplementedException` when called